cmd/tuf/app/add-delegation.go

```diff
@@ -102,7 +102,7 @@ func DelegationCmd(ctx context.Context, directory, name, path string, terminatin
 	keys := []*data.PublicKey{}
 	ids := []string{}
 	for _, keyRef := range keyRefs {
-		signerKey, err := pkeys.GetSigningKey(ctx, keyRef)
+		signerKey, err := pkeys.GetSigningKey(ctx, keyRef, true)
```

Comment:
nit: Either create a named boolean var or add /* deprecated */ where `true/false` are used, helps with readability

Reply:
This is a great idea and I am embarassed! :D